### PR TITLE
add provider-gcp and provider-baidu in consume-cloud-services.md

### PR DESCRIPTION
**Pull request by AveryQi115**
Signed-off-by: Nicola115 <2225992901@qq.com>

### CODE REVIEW

File: docs/tutorials/consume-cloud-services.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ First, visit [VelaUX (KubeVela Dashboard)](../install#3-install-velaux), switch
 - provider-alibaba
 - provider-aws
 - provider-azure
-- provider-tencent
+- provider-tencent  
+- provider-gcp  
+- provider-baidu
```
Comment (zzxwill) [inline docs/tutorials/consume-cloud-services.md]:
```suggestion
- terraform-tencent  
- terraform-gcp  
- terraform-baidu  
```
Please also update others.

File: versioned_docs/version-v1.2/tutorials/consume-cloud-services.md [hunk 2]
```diff
@@ -25,7 +25,9 @@ First, visit [VelaUX (KubeVela Dashboard)](../install#3-install-velaux), switch
 - provider-alibaba
 - provider-aws
 - provider-azure
-- provider-tencent
+- provider-tencent  
+- provider-gcp  
+- provider-baidu  
 
 We can provision cloud resources in Alibaba Cloud, AWS, Azure and Tencent Cloud respectively.
```
Comment (zzxwill) [inline versioned_docs/version-v1.2/tutorials/consume-cloud-services.md]:
```suggestion
We can provision cloud resources in Alibaba Cloud, AWS, Azure, Tencent Cloud, Goole Cloud Platform and Baidu Cloud respectively.
```

File: docs/tutorials/consume-cloud-services.md [hunk 3]
```diff
@@ -25,11 +25,11 @@ First, visit [VelaUX (KubeVela Dashboard)](../install#3-install-velaux), switch
 - provider-alibaba
```
Comment (zzxwill) [inline docs/tutorials/consume-cloud-services.md]: These should also be updated:)

Comment (AveryQi115) [reply docs/tutorials/consume-cloud-services.md]: should all the provider-xxx be replaced as terraform-xxx? or just provider-alibaba?

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/consume-cloud-services.md [hunk 4]
```diff
@@ -20,12 +20,14 @@ description: 本文介绍通过 KubeVela 交付云厂商提供的服务，并能
...
-- provider-tencent
+- terraform-alibaba
+- terraform-aws
+- terraform-azure
+- terraform-tencent  
+- terraform-gcp  
+- terraform-baidu  
 
-我们可以分别在阿里巴巴云、AWS、Azure 和腾讯云配置云资源。
+我们可以分别在阿里巴巴云、AWS、Azure、腾讯云、Google Cloud Platform和百度云配置云资源。
```
Comment (zzxwill) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/consume-cloud-services.md]:
```suggestion
我们可以分别在阿里云、AWS、Azure、腾讯云、Google Cloud Platform和百度云配置云资源。
```
I might mistakenly translate `Alibaba Cloud`, sorry.

**Review by zzxwill — APPROVED** [review]

**Pull request merged**